react-native-fbsdklogin/js/FBSDKLoginManager.ios.js

```diff
@@ -51,63 +57,52 @@ type FBSDKDefaultAudience = $Enum<{
   'only-me': string,
 }>;
 
-/**
- * Provides methods for logging the user in and out.
- */
-class FBSDKLoginManager {
+module.exports = {
```

Comment:
Yeah, once I peeled away all that unnecessary syntactic sugar, I discovered the same. I had the same thought, but didn't know how to do it. How do you just simply do
```js
module.exports = require('react-native').NativeModules.FBSDKLoginManager;
```
while also providing Flow definitions?